### PR TITLE
feat: Cerebras Support

### PR DESCRIPTION
**Pull request by tcapelle**
Adds support for Cerebras API.
- Currently supports sync and async non-streamed messages
- It needs to change the cerebras SDK install to pypi once release

### CODE REVIEW

File: requirements.test.txt [hunk 1]
```diff
@@ -43,6 +43,7 @@ chromadb>=0.5.0             # LangChain
 pysqlite3-binary==0.5.3     # LangChain
 cohere>=5.5.8               # Cohere
 groq>=0.9.0                 # Groq
-
+# cerebras-cloud-sdk          # cerebras-cloud-sdk
+cerebras-cloud-sdk @ https://cerebras-cloud-sdk.s3.amazonaws.com/production/cerebras_cloud_sdk-0.7.0-py3-none-any.whl
```
Comment (andrewtruong) [inline requirements.test.txt:47]: remove

File: weave/integrations/cerebras/cassettes/cerebras_test/test_cerebras_async.yaml
Comment (andrewtruong) [inline weave/integrations/cerebras/cassettes/cerebras_test/test_cerebras_async.yaml:1]: is there async support in this PR?  Seems like you removed it from the docs

File: weave/integrations/cerebras/cerebras_sdk.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            ) -> typing.Any:
+                return await fn(*args, **kwargs)
+
+            return _async_wrapper
+
+        op = weave.op()(_fn_wrapper(fn))
+        op.name = name  # type: ignore
+        return op
+
+    return wrapper
```
Comment (andrewtruong) [inline weave/integrations/cerebras/cerebras_sdk.py:37]: is there async support in this PR?  Seems like you removed it from the docs

File: weave/integrations/cerebras/cerebras_sdk.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        SymbolPatcher(
+            lambda: importlib.import_module("cerebras.cloud.sdk.resources.chat"),
+            "CompletionsResource.create",
+            create_wrapper_sync(name="cerebras.chat.completions.create"),
+        ),
+        SymbolPatcher(
+            lambda: importlib.import_module("cerebras.cloud.sdk.resources.chat"),
+            "AsyncCompletionsResource.create",
+            create_wrapper_async(name="cerebras.chat.completions.create"),
+        ),
```
Comment (andrewtruong) [inline weave/integrations/cerebras/cerebras_sdk.py:51]: is there async support in this PR?  Seems like you removed it from the docs

File: requirements.test.txt (hunk 1 above)
Comment (tcapelle) [reply requirements.test.txt:47]: not yet =P

**Comment by tcapelle** [comment]
> Why did you remove async from the docs? It seems like it's still in the PR

Cause we don't have that differentiation on other integrations

File: docs/docs/guides/integrations/cerebras.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            messages=[{"role": "user", "content": f"What's the top speed of a {animal}?"}],
+            temperature=self.temperature
+        )
+        return response.choices[0].message.content
+
+speed_model = AnimalSpeedModel(
+    model="llama3.1-8b",
+    temperature=0.7
+)
+result = speed_model.predict(animal="peregrine falcon")
```
Comment (andrewtruong) [inline docs/docs/guides/integrations/cerebras.md:101]: the picture shows "cheetah", but you have "peregrine falcon" here

**Pull request merged**